src/components/ModelTableList/StatusGroup.test.js

```diff
@@ -75,4 +86,23 @@ describe("StatusGroup", () => {
       .first();
     expect(logo.prop("src")).toContain("gce");
   });
+
+  it("model access buttons are present in status group", () => {
+    const store = mockStore(dataDump);
+    const filters = {
+      cloud: ["aws"],
+    };
+    const wrapper = mount(
+      <MemoryRouter>
+        <Provider store={store}>
+          <QueryParamProvider ReactRouterRoute={Route}>
+            <StatusGroup filters={filters} />
+          </QueryParamProvider>
+        </Provider>
+      </MemoryRouter>
+    );
+    const firstContentRow = wrapper.find(".status-group tr").at(1);
+    const modelAccessButton = firstContentRow.find(".model-access");
+    expect(modelAccessButton.length).toBe(2);
```

Comment:
Can you add one more assertion that both of the columns are there? One for mobile and desktop?